google/cloud/storage/tests/testbench.py

```diff
@@ -339,30 +339,46 @@ class GcsBucket(object):
 
     def __init__(self, gcs_url, name):
         self.name = name
+        self.gcs_url = gcs_url
         self.metadata = {
-            'id': name,
-            'kind': 'storage#bucket',
-            'metageneration': '4',
-            'selfLink': gcs_url + name,
-            'projectNumber': '123456789',
-            'name': name,
-            'timeCreated': '2018-05-19T19:31:14Z',
-            'updated': '2018-05-19T19:31:24Z',
+            'metageneration': 0,
+            'name': self.name,
             'location': 'US',
             'storageClass': 'STANDARD',
             'etag': 'XYZ=',
             'labels': {
-                'foo': 'bar',
-                'baz': 'qux'
-            }
+                 'foo': 'bar',
```

Comment:
Looks like the added spaces were unnecessary (and makes this 5 instead of 4)... not that we currently have a linter, but meh.